UnitTests/UnitTests.NetStandard/SampleModelWithINPCAndObservableProperties.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Toolkit.Mvvm.ComponentModel;
+
+namespace UnitTests.NetStandard
+{
+    /// <summary>
+    /// See https://github.com/CommunityToolkit/WindowsCommunityToolkit/issues/3665.
```

Comment:
Same here? https://github.com/CommunityToolkit/WindowsCommunityToolkit/issues/4167

Reply:
Same as above, fixed in 5c1b35bd3fd45f34a1f626f5ea54c18d9354e1d8 🙂